types/tabulator-tables/index.d.ts

```diff
@@ -2862,7 +2862,7 @@ declare class Tabulator {
     /** Destructor. */
     destroy: () => void;
 
-    setData: (data: any, params?: any, config?: any) => Promise<void>;
+    setData: (data?: any, params?: any, config?: any) => Promise<void>;
```

Comment:
Why's that necessary? `any` includes `undefined`.

Reply:
Turns out `any` (or `unknown` for that matter) is not the same as an optional parameter.

Reply:
I didn't add `any`, it was there before my PR. I just added `?` to make the first parameter optional. However, `?` will make the parameter optional and nullish, it has no relation with the data type.